[ 2024-01-02 20:27:13,620,620 ]  [ INFO     ]  [ MainThread  ]  [ do_mysql.py / 77 / __init__ ] -- ->  < id: test > 连接 MySql 成功
[ 2024-01-02 20:27:13,678,678 ]  [ INFO     ]  [ MainThread  ]  [ do_mysql.py / 93 / __del__ ] -- ->  < id: test > 断开 MySql 成功
[ 2024-01-03 14:38:18,457,457 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 19 / _all ] -- -> ----执行开始----
[ 2024-01-03 14:38:18,473,473 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 21 / _all ] -- -> ----执行完毕----
[ 2024-01-03 14:38:52,020,020 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 14:38:52,033,033 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 14:43:42,923,923 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 14:43:42,935,935 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 16:41:26,197,197 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 16:41:26,223,223 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 16:42:08,730,730 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 16:42:08,738,738 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 16:42:46,036,036 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 16:42:46,044,044 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 16:42:57,070,070 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 16:42:57,083,083 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 17:12:38,868,868 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 17:12:38,880,880 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_create_customer[login0]
[ 2024-01-03 17:12:38,975,975 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 17:13:34,996,996 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 17:16:22,023,023 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 17:16:26,365,365 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 17:16:26,366,366 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_create_customer[login0]
[ 2024-01-03 17:16:26,366,366 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 17:16:26,366,366 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 17:16:26,366,366 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 17:16:26,366,366 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.0001627000019652769
[ 2024-01-03 17:16:26,366,366 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 17:16:26,370,370 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_create_customer[login1]
[ 2024-01-03 17:16:26,370,370 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 17:16:26,370,370 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 17:16:26,370,370 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 17:16:26,371,371 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.0002018999948631972
[ 2024-01-03 17:16:26,371,371 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 17:16:26,374,374 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_create_customer[login2]
[ 2024-01-03 17:16:26,375,375 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 17:16:26,375,375 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 17:16:26,375,375 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 17:16:26,375,375 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.00017129999469034374
[ 2024-01-03 17:16:26,375,375 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 17:16:26,378,378 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_cooperation[login0]
[ 2024-01-03 17:16:26,378,378 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 17:16:26,379,379 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 17:16:26,379,379 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 17:16:26,379,379 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.00012680000509135425
[ 2024-01-03 17:16:26,379,379 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 17:16:26,382,382 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_cooperation[login1]
[ 2024-01-03 17:16:26,382,382 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 17:16:26,382,382 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 17:16:26,382,382 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 17:16:26,382,382 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.00013299999409355223
[ 2024-01-03 17:16:26,383,383 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 17:16:26,386,386 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_cooperation[login2]
[ 2024-01-03 17:16:26,386,386 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 17:16:26,386,386 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 17:16:26,387,387 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 17:16:26,387,387 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.0003220000071451068
[ 2024-01-03 17:16:26,387,387 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 17:16:26,388,388 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:54:03,827,827 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:54:03,846,846 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:54:45,923,923 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:54:45,930,930 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:55:06,115,115 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:55:06,122,122 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:55:25,529,529 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:55:25,538,538 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:55:40,821,821 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:55:40,829,829 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:55:52,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:55:52,467,467 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:55:56,148,148 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:55:56,158,158 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:56:04,458,458 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:56:04,468,468 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:56:16,425,425 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:56:16,432,432 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:56:25,828,828 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:56:25,836,836 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 18:56:35,992,992 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 20 / _all ] -- -> ----执行开始----
[ 2024-01-03 18:56:35,993,993 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_create_customer[login10]
[ 2024-01-03 18:56:35,993,993 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 18:56:35,993,993 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 18:56:35,993,993 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 18:56:35,993,993 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.00013899999612476677
[ 2024-01-03 18:56:35,994,994 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 18:56:35,997,997 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_create_customer[login11]
[ 2024-01-03 18:56:35,997,997 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 18:56:35,997,997 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 18:56:35,997,997 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 18:56:35,998,998 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.0001429999974789098
[ 2024-01-03 18:56:35,998,998 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 18:56:36,000,000 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_create_customer[login12]
[ 2024-01-03 18:56:36,000,000 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 18:56:36,000,000 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 18:56:36,000,000 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 18:56:36,001,001 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.0001444000081392005
[ 2024-01-03 18:56:36,001,001 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 18:56:36,003,003 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_cooperation[login10]
[ 2024-01-03 18:56:36,003,003 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 18:56:36,003,003 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 18:56:36,003,003 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 18:56:36,003,003 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.00012959999730810523
[ 2024-01-03 18:56:36,004,004 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 18:56:36,007,007 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_cooperation[login11]
[ 2024-01-03 18:56:36,007,007 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 18:56:36,007,007 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 18:56:36,007,007 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 18:56:36,007,007 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.00012900000729132444
[ 2024-01-03 18:56:36,007,007 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 18:56:36,010,010 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestCRM::test_cooperation[login12]
[ 2024-01-03 18:56:36,010,010 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 68 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-03 18:56:36,010,010 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 69 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-03 18:56:36,010,010 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-03 18:56:36,010,010 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.00014119999832473695
[ 2024-01-03 18:56:36,010,010 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-03 18:56:36,012,012 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 22 / _all ] -- -> ----执行完毕----
[ 2024-01-03 19:55:31,107,107 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row_number': 3}
[ 2024-01-03 19:55:31,812,812 ]  [ ERROR    ]  [ MainThread  ]  [ base_assert.py / 109 / assert_common ] -- -> 断言不通过 : USER_NOT_FOUND != USER_LOGIN_INFO_NOT_FOUND
[ 2024-01-03 19:55:31,922,922 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row_number': 4}
[ 2024-01-03 19:55:32,688,688 ]  [ ERROR    ]  [ MainThread  ]  [ base_assert.py / 109 / assert_common ] -- -> 断言不通过 : PASSWORD_ERROR != OK
[ 2024-01-03 19:55:32,699,699 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row_number': 5}
[ 2024-01-03 19:55:33,529,529 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-03 19:58:26,735,735 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row_number': 3}
[ 2024-01-03 19:58:27,414,414 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'c4904c2c-65ae-482d-93d5-817bc2aee0cf', 'message': '当前账号不存在'}
[ 2024-01-03 19:58:27,415,415 ]  [ ERROR    ]  [ MainThread  ]  [ base_assert.py / 109 / assert_common ] -- -> 断言不通过 : USER_NOT_FOUND != USER_LOGIN_INFO_NOT_FOUND
[ 2024-01-03 19:58:27,512,512 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row_number': 4}
[ 2024-01-03 19:58:28,231,231 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:SFm8kUMu69Io4k5AUpds71JcofSckGnoT8cel4LYD2k=129babd8e1e44245a189abee69c93ed1'}, 'application': 'yl-yft-api', 'traceId': '4a199098a9f145c5aa6f5d73bed458ed', 'message': 'OK'}
[ 2024-01-03 19:58:28,232,232 ]  [ ERROR    ]  [ MainThread  ]  [ base_assert.py / 109 / assert_common ] -- -> 断言不通过 : PASSWORD_ERROR != OK
[ 2024-01-03 19:58:28,240,240 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row_number': 5}
[ 2024-01-03 19:58:28,980,980 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:5alfZItWJpN4fE03ThsEfVncAN9uLhh/hkeDu7zi+bs=a443eb728e9c46ac8ecbd39f24328031'}, 'application': 'yl-yft-api', 'traceId': '3bc177f690cd49009f3f7ad02374a7e4', 'message': 'OK'}
[ 2024-01-03 19:58:28,981,981 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-03 20:00:53,570,570 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row_number': 3}
[ 2024-01-03 20:00:54,295,295 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '7ffab752-8e0a-403e-821b-0cb7bee467ca', 'message': '当前账号不存在'}
[ 2024-01-03 20:00:54,295,295 ]  [ ERROR    ]  [ MainThread  ]  [ base_assert.py / 109 / assert_common ] -- -> 断言不通过 : USER_NOT_FOUND != USER_LOGIN_INFO_NOT_FOUND
[ 2024-01-03 20:00:54,389,389 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row_number': 4}
[ 2024-01-03 20:00:55,104,104 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'c61e5cab-521a-4ed7-8ba5-aa190a37b23f', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-03 20:00:55,105,105 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : PASSWORD_ERROR == PASSWORD_ERROR
[ 2024-01-03 20:00:55,106,106 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row_number': 5}
[ 2024-01-03 20:00:55,827,827 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:0/q3ErKmaErfEHZuTPonWGNlNINVC40wHjRjTPhp12A=8443f77048d04a1ea50f094527f8cf72'}, 'application': 'yl-yft-api', 'traceId': 'a664d7af10ae4a6c854ad4a3e3dcebcc', 'message': 'OK'}
[ 2024-01-03 20:00:55,827,827 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-03 20:02:23,641,641 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row_number': 3}
[ 2024-01-03 20:02:24,315,315 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'f82de2be-1a1a-4790-b6ba-07fbf2fd8370', 'message': '当前账号不存在'}
[ 2024-01-03 20:02:24,316,316 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-03 20:02:24,317,317 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row_number': 4}
[ 2024-01-03 20:02:25,051,051 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '2448c1d3-abc3-4e94-8b1c-fa224102d19c', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-03 20:02:25,051,051 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-03 20:02:25,056,056 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row_number': 5}
[ 2024-01-03 20:02:25,770,770 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:ZMOlbW5feMz72o20SmsOsWw5qmOPP/LRzF2Q/j9BR4E=70e5e91ecf6444059bce66d466ded0a0'}, 'application': 'yl-yft-api', 'traceId': '92f5a3f730aa43b7a04ca1f268b526cd', 'message': 'OK'}
[ 2024-01-03 20:02:25,770,770 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-03 22:00:00,159,159 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row_number': 3}
[ 2024-01-03 22:00:00,362,362 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '5b9accdf-a556-4d4f-9658-2dc7623d948d', 'message': '当前账号不存在'}
[ 2024-01-03 22:00:00,362,362 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-03 22:00:00,362,362 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row_number': 4}
[ 2024-01-03 22:00:00,507,507 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '9a7755ac-4d2b-4d98-a25a-5a05d8242afb', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-03 22:00:00,522,522 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-03 22:00:00,522,522 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row_number': 5}
[ 2024-01-03 22:00:00,713,713 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_create_customer ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:QzTddBYaIkXum2en88qQEXn4k42VKCJp9m/OuRngdAc=f748e8c99b924e78933290a2c92f9cb3'}, 'application': 'yl-yft-api', 'traceId': '711222e0d2be4c2d856f4c9712b33327', 'message': 'OK'}
[ 2024-01-03 22:00:00,713,713 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-03 22:00:00,713,713 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '8900', 'verifyId': 'cdb135f4278c4fef97865a34f2f239e3'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row_number': 3}
[ 2024-01-03 22:00:00,775,775 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '5817', 'verifyId': '2cfc99e9a8e2490b880401214ba5f5db'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row_number': 4}
[ 2024-01-03 22:00:00,854,854 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 24 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '3724', 'verifyId': 'bede6d5dda4a43c49feddfe781a4c82c'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row_number': 5}
[ 2024-01-03 23:39:51,754,754 ]  [ ERROR    ]  [ MainThread  ]  [ do_try.py / 65 / __exit__ ] -- -> 'b'
Traceback (most recent call last):
  File "D:/Useless/Desktop/yljk_test_api/common/utils/do_try.py", line 84, in <module>
    var2 = {'a': 'aaa'}['b']
KeyError: 'b'
[ 2024-01-04 00:21:37,606,606 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row_number': 3}
[ 2024-01-04 00:21:37,853,853 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '0a585cea-ce1f-445f-841d-8bb543bb692a', 'message': '当前账号不存在'}
[ 2024-01-04 00:21:37,853,853 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:21:37,861,861 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 51 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login0]
[ 2024-01-04 00:21:37,861,861 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:21:37,862,862 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-04 00:21:37,862,862 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-04 00:21:37,863,863 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 73 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.11608950000000018
[ 2024-01-04 00:21:37,863,863 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 74 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-04 00:21:37,873,873 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row_number': 4}
[ 2024-01-04 00:21:38,121,121 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'aae7eb1c-00f7-49c1-be3b-b930310e1156', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:21:38,121,121 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:21:38,121,121 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 51 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login1]
[ 2024-01-04 00:21:38,121,121 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:21:38,121,121 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-04 00:21:38,123,123 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-04 00:21:38,123,123 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 73 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.1284453000000001
[ 2024-01-04 00:21:38,124,124 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 74 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-04 00:21:38,134,134 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row_number': 5}
[ 2024-01-04 00:21:38,396,396 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:rWPcuMtPOeROdqceyTbTZh8mSOUV5UWt18IV7prTmHc=35c195da784b4fc086bf59ed42ab20e3'}, 'application': 'yl-yft-api', 'traceId': '53f9e583ef264858b7836284ef4c19b3', 'message': 'OK'}
[ 2024-01-04 00:21:38,398,398 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:21:38,399,399 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 51 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login2]
[ 2024-01-04 00:21:38,400,400 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 70 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:21:38,400,400 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 71 / pytest_runtest_makereport ] -- -> 故障表示 : None
[ 2024-01-04 00:21:38,401,401 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 72 / pytest_runtest_makereport ] -- -> 异常 : None
[ 2024-01-04 00:21:38,401,401 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 73 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.15301429999999971
[ 2024-01-04 00:21:38,402,402 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 74 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-04 00:28:09,743,743 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:28:09,908,908 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'ff0c5ef1-9451-4cc3-863b-98febdecd037', 'message': '当前账号不存在'}
[ 2024-01-04 00:28:09,908,908 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:28:09,908,908 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login0]
[ 2024-01-04 00:28:09,908,908 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 56 / pytest_runtest_makereport ] -- -> 用例位置 : 登录模块 第 3 行
[ 2024-01-04 00:28:26,294,294 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:28:26,458,458 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'f9200d14-3f6d-493c-8e04-e0cd196ce3a9', 'message': '当前账号不存在'}
[ 2024-01-04 00:28:26,458,458 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:28:26,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login0]
[ 2024-01-04 00:28:26,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 56 / pytest_runtest_makereport ] -- -> 用例位置 : 登录模块 第 3 行
[ 2024-01-04 00:28:26,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 57 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:28:26,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 58 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:28:26,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 59 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '1999', 'verifyId': 'c8ca39dddcde45b499cd401a1cb58272'}
[ 2024-01-04 00:28:26,460,460 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 62 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:28:26,460,460 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 63 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:28:26,460,460 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 64 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.0988093000000001
[ 2024-01-04 00:28:26,461,461 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 65 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-04 00:28:26,466,466 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:28:26,650,650 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'e0a861d0-c678-493a-9f5f-50a7cf15bfed', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:28:26,650,650 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:28:26,651,651 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login1]
[ 2024-01-04 00:28:26,651,651 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 56 / pytest_runtest_makereport ] -- -> 用例位置 : 登录模块 第 4 行
[ 2024-01-04 00:28:26,651,651 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 57 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:28:26,651,651 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 58 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:28:26,651,651 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 59 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '3837', 'verifyId': 'f2e1c80c036242dcbffa5ee9e931f988'}
[ 2024-01-04 00:28:26,651,651 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 62 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:28:26,651,651 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 63 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:28:26,652,652 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 64 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.10468920000000015
[ 2024-01-04 00:28:26,652,652 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 65 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-04 00:28:26,656,656 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:28:26,886,886 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:SUCEEa4qDJRT6Ha7Vn4oTP4+vPqYoz08JfDckkI5B54=5be61250e85243cfadcb0b1439a88a12'}, 'application': 'yl-yft-api', 'traceId': '964ea3f164f64eda91ee2f56edd40ead', 'message': 'OK'}
[ 2024-01-04 00:28:26,886,886 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:28:26,887,887 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- -> 用例ID : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login2]
[ 2024-01-04 00:28:26,887,887 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 56 / pytest_runtest_makereport ] -- -> 用例位置 : 登录模块 第 5 行
[ 2024-01-04 00:28:26,887,887 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 57 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:28:26,887,887 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 58 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:28:26,888,888 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 59 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '8402', 'verifyId': '2a326ed83fd844788bf97f373b01d124'}
[ 2024-01-04 00:28:26,888,888 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 62 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:28:26,888,888 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 63 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:28:26,888,888 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 64 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.14358389999999988
[ 2024-01-04 00:28:26,888,888 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 65 / pytest_runtest_makereport ] -- -> ****************************************
[ 2024-01-04 00:31:13,302,302 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:31:13,495,495 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '2a8ca359-b618-45d8-b647-f5592eac6907', 'message': '当前账号不存在'}
[ 2024-01-04 00:31:13,496,496 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:31:13,497,497 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login0]
[ 2024-01-04 00:34:35,842,842 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:34:36,084,084 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '183cfd86-1774-4a55-8448-54715293ecf5', 'message': '当前账号不存在'}
[ 2024-01-04 00:34:36,084,084 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:34:36,085,085 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login0]
[ 2024-01-04 00:35:29,764,764 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:35:30,001,001 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'e3d50d05-68ce-440c-b262-bbfb3609c4bf', 'message': '当前账号不存在'}
[ 2024-01-04 00:35:30,001,001 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:35:30,003,003 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login0]
[ 2024-01-04 00:35:30,003,003 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <3> 行
[ 2024-01-04 00:35:30,004,004 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:35:30,004,004 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:35:30,005,005 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '1201', 'verifyId': 'd9b95c534e8c4d409549c2181770ce1c'}
[ 2024-01-04 00:35:30,005,005 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:35:30,006,006 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:35:30,006,006 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.12092390000000108
[ 2024-01-04 00:35:30,007,007 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_< 
[ 2024-01-04 00:35:30,027,027 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:35:30,238,238 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '754d35ba-d099-4a68-8c8e-568f13f6dbf5', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:35:30,239,239 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:35:30,239,239 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login1]
[ 2024-01-04 00:35:30,240,240 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <4> 行
[ 2024-01-04 00:35:30,240,240 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:35:30,241,241 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:35:30,241,241 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '8396', 'verifyId': '37f2597314074b7fa6011294fa6c033d'}
[ 2024-01-04 00:35:30,241,241 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:35:30,242,242 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:35:30,242,242 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.11466120000000046
[ 2024-01-04 00:35:30,243,243 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_< 
[ 2024-01-04 00:35:30,253,253 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:35:30,482,482 ]  [ INFO     ]  [ MainThread  ]  [ test_create_customer.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:kp4Yh6IVMPJxBQ3TligLQAdW4yTQFDzn1Hbd3WQc1Eg=c2966d1bd0e34802a6d25fb8d3a378cb'}, 'application': 'yl-yft-api', 'traceId': '4611c4738c1c4cfdae69bb530097d6c0', 'message': 'OK'}
[ 2024-01-04 00:35:30,482,482 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:35:30,483,483 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_crm/test_create_customer.py::TestLogin::test_login[login2]
[ 2024-01-04 00:35:30,483,483 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <5> 行
[ 2024-01-04 00:35:30,484,484 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:35:30,484,484 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:35:30,484,484 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '6470', 'verifyId': '3326fdb3371b4d7684f029673520af42'}
[ 2024-01-04 00:35:30,485,485 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:35:30,485,485 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:35:30,485,485 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.13780319999999868
[ 2024-01-04 00:35:30,486,486 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_< 
[ 2024-01-04 00:36:56,096,096 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:36:56,279,279 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'db120f38-301f-440a-b16c-a7c73211e004', 'message': '当前账号不存在'}
[ 2024-01-04 00:36:56,279,279 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:36:56,280,280 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login0]
[ 2024-01-04 00:36:56,280,280 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <3> 行
[ 2024-01-04 00:36:56,280,280 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:36:56,280,280 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:36:56,281,281 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '6209', 'verifyId': '6aaa009dcd4a40748aba0a52db94f7c8'}
[ 2024-01-04 00:36:56,281,281 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:36:56,281,281 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:36:56,281,281 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.09989740000000014
[ 2024-01-04 00:36:56,282,282 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_< 
[ 2024-01-04 00:36:56,287,287 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:36:56,475,475 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'a4f71568-9e90-42d5-895e-5e472eb933fb', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:36:56,476,476 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:36:56,477,477 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login1]
[ 2024-01-04 00:36:56,477,477 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <4> 行
[ 2024-01-04 00:36:56,477,477 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:36:56,478,478 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:36:56,478,478 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '7021', 'verifyId': 'f68d46d29c034edca45464ee74eda2fe'}
[ 2024-01-04 00:36:56,478,478 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:36:56,478,478 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:36:56,479,479 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.10967939999999965
[ 2024-01-04 00:36:56,479,479 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_< 
[ 2024-01-04 00:36:56,486,486 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:36:56,721,721 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:yCs4AcLb2Gw/VajqFAjhGKuAr7tN3Muodk1oI0cJNSw=7aeb9faff8be47fc928a32a1920d8ea1'}, 'application': 'yl-yft-api', 'traceId': '032e3c9b436d4fbaa9da7572eb73b7b9', 'message': 'OK'}
[ 2024-01-04 00:36:56,721,721 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:36:56,722,722 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login2]
[ 2024-01-04 00:36:56,722,722 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <5> 行
[ 2024-01-04 00:36:56,722,722 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:36:56,722,722 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:36:56,722,722 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '3274', 'verifyId': '0d78d26da8a64e1682648dac67e3d28e'}
[ 2024-01-04 00:36:56,722,722 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:36:56,722,722 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:36:56,723,723 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.14102820000000005
[ 2024-01-04 00:36:56,723,723 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_<  >_< 
[ 2024-01-04 00:38:01,175,175 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:38:01,362,362 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '9db6ddc8-ca4e-4600-8ecb-563647d223b6', 'message': '当前账号不存在'}
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login0]
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <3> 行
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '9268', 'verifyId': '9470c8b1cd7442129c52b9cdbb3b85ea'}
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:38:01,363,363 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:38:01,364,364 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.10073259999999973
[ 2024-01-04 00:38:01,364,364 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> *_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_*
[ 2024-01-04 00:38:01,368,368 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:38:01,548,548 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '5227b644-5057-46a6-85f1-668060b36806', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:38:01,549,549 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:38:01,549,549 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login1]
[ 2024-01-04 00:38:01,549,549 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <4> 行
[ 2024-01-04 00:38:01,549,549 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:38:01,549,549 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:38:01,550,550 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '3837', 'verifyId': 'f380c4c7e12c40b7ad83dcc16b195a6b'}
[ 2024-01-04 00:38:01,550,550 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:38:01,550,550 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:38:01,550,550 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.10040039999999983
[ 2024-01-04 00:38:01,550,550 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> *_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_*
[ 2024-01-04 00:38:01,555,555 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:38:01,796,796 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:6TcJ5II+Lz0ZdWObGQxAne4Affki9gnfkg1Zmcx7gzE=b0ba745edca04d058854f2d2cc8b7e8d'}, 'application': 'yl-yft-api', 'traceId': '709f05dd50164bd695393cffc275039f', 'message': 'OK'}
[ 2024-01-04 00:38:01,796,796 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:38:01,797,797 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login2]
[ 2024-01-04 00:38:01,797,797 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <5> 行
[ 2024-01-04 00:38:01,797,797 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:38:01,798,798 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:38:01,798,798 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '5684', 'verifyId': '12d7ab704a974797816870a3915d5b3f'}
[ 2024-01-04 00:38:01,798,798 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:38:01,798,798 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:38:01,798,798 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.14937500000000004
[ 2024-01-04 00:38:01,798,798 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> *_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_**_*
[ 2024-01-04 00:38:26,888,888 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:38:27,038,038 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '9a1599d2-39c2-4903-805a-6a92ff8c04aa', 'message': '当前账号不存在'}
[ 2024-01-04 00:38:27,039,039 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:38:27,040,040 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login0]
[ 2024-01-04 00:38:27,041,041 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <3> 行
[ 2024-01-04 00:38:27,041,041 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:38:27,042,042 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:38:27,042,042 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '8821', 'verifyId': 'b5ed6933fe2f4f07a4977b032ba242d2'}
[ 2024-01-04 00:38:27,042,042 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:38:27,043,043 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:38:27,043,043 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.08668560000000003
[ 2024-01-04 00:38:27,043,043 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:38:27,050,050 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:38:27,258,258 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'afb88490-884f-4712-b033-0344286a9606', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:38:27,258,258 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:38:27,259,259 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login1]
[ 2024-01-04 00:38:27,259,259 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <4> 行
[ 2024-01-04 00:38:27,259,259 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:38:27,260,260 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:38:27,260,260 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '6683', 'verifyId': 'fc6a957488b842cf9299d806382b8b9a'}
[ 2024-01-04 00:38:27,260,260 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:38:27,260,260 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:38:27,260,260 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.11653199999999986
[ 2024-01-04 00:38:27,261,261 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:38:27,267,267 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:38:27,463,463 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:XsyHP6lOR/nEukitYexEYUpn5LrVOm7u9+t1nLCO2oQ=1b0731dd67af450ba56a102ab7eecf63'}, 'application': 'yl-yft-api', 'traceId': '743a2884a3844376a14e8ee051283906', 'message': 'OK'}
[ 2024-01-04 00:38:27,464,464 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:38:27,464,464 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 39 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login2]
[ 2024-01-04 00:38:27,464,464 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 42 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <5> 行
[ 2024-01-04 00:38:27,464,464 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:38:27,465,465 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:38:27,465,465 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '9863', 'verifyId': '7700f09833494468be05db54e0817eaf'}
[ 2024-01-04 00:38:27,465,465 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:38:27,465,465 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:38:27,465,465 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.12507480000000015
[ 2024-01-04 00:38:27,465,465 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:39:53,020,020 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / pytest_collection_modifyitems ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:39:53,020,020 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / pytest_collection_modifyitems ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:39:53,021,021 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / pytest_collection_modifyitems ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:39:53,109,109 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:39:53,283,283 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '77c4af57-8133-45ee-b14c-3e7827dd41ff', 'message': '当前账号不存在'}
[ 2024-01-04 00:39:53,284,284 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:39:53,284,284 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login0]
[ 2024-01-04 00:39:53,284,284 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <3> 行
[ 2024-01-04 00:39:53,284,284 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:39:53,284,284 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:39:53,284,284 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '7446', 'verifyId': 'b18799a8d2824ae686602947bda0f96a'}
[ 2024-01-04 00:39:53,284,284 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:39:53,285,285 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:39:53,285,285 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.10598980000000013
[ 2024-01-04 00:39:53,285,285 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:39:53,289,289 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:39:53,457,457 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'c09f658a-64ba-4609-9edb-6c2e5f6a3b66', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:39:53,458,458 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:39:53,458,458 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login1]
[ 2024-01-04 00:39:53,458,458 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <4> 行
[ 2024-01-04 00:39:53,458,458 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:39:53,458,458 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:39:53,458,458 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '1755', 'verifyId': '45ef0530aa6346418f04e85abdfae820'}
[ 2024-01-04 00:39:53,458,458 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:39:53,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:39:53,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.08265649999999969
[ 2024-01-04 00:39:53,459,459 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:39:53,464,464 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:39:53,689,689 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:5PdvrAqP4ydEDiImE3AgbUpy8ksbscXfR2Pr6146OKs=1bc32697df2946efaf4733e9da6d7d4b'}, 'application': 'yl-yft-api', 'traceId': '44b7ca825faf4b57bfaf9ae6330b462e', 'message': 'OK'}
[ 2024-01-04 00:39:53,689,689 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:39:53,689,689 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login2]
[ 2024-01-04 00:39:53,689,689 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <5> 行
[ 2024-01-04 00:39:53,689,689 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:39:53,690,690 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:39:53,690,690 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '5579', 'verifyId': '2f15acc3365049309a3e9bddd0c3a8a3'}
[ 2024-01-04 00:39:53,690,690 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:39:53,690,690 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:39:53,690,690 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.1467170000000002
[ 2024-01-04 00:39:53,690,690 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,430,430 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:40:28,496,496 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,593,593 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'f194d2bd-60e8-47df-a067-4326021b412b', 'message': '当前账号不存在'}
[ 2024-01-04 00:40:28,594,594 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:40:28,594,594 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,595,595 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login0]
[ 2024-01-04 00:40:28,595,595 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <3> 行
[ 2024-01-04 00:40:28,595,595 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:40:28,595,595 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:40:28,595,595 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '1037', 'verifyId': 'b1630233576a43cf9dc5d9cf4c9db683'}
[ 2024-01-04 00:40:28,595,595 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:40:28,596,596 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:40:28,596,596 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.09760630000000026
[ 2024-01-04 00:40:28,596,596 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,598,598 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,604,604 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:40:28,705,705 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,806,806 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'cc10d4d2-bd89-4838-81fd-feff64c871f5', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:40:28,806,806 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:40:28,807,807 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,807,807 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login1]
[ 2024-01-04 00:40:28,807,807 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <4> 行
[ 2024-01-04 00:40:28,807,807 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:40:28,807,807 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:40:28,807,807 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '6723', 'verifyId': '08490e3e719f44ddaee0b3a7d64246f3'}
[ 2024-01-04 00:40:28,807,807 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:40:28,808,808 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:40:28,808,808 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.10101369999999976
[ 2024-01-04 00:40:28,808,808 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,810,810 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:28,814,814 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:40:28,903,903 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:29,024,024 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:dZhOOevGvrk2sWQ1rEaoUaRJshiPocz/ZqCZCWC6bRY=3b1b64081c1c4ea5b82e86866ff2cba5'}, 'application': 'yl-yft-api', 'traceId': '5cf00dee24984c0faac8b432c6cadd63', 'message': 'OK'}
[ 2024-01-04 00:40:29,024,024 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:40:29,024,024 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:29,024,024 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login2]
[ 2024-01-04 00:40:29,025,025 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <5> 行
[ 2024-01-04 00:40:29,025,025 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:40:29,025,025 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:40:29,025,025 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '5188', 'verifyId': '0de53aab033a441e9a8aaffe5a177f91'}
[ 2024-01-04 00:40:29,025,025 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:40:29,025,025 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:40:29,025,025 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.12130829999999992
[ 2024-01-04 00:40:29,026,026 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:40:29,029,029 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:49,619,619 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login001', 'title': '用户名错误，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'da00991470fe444cb01f8754c2c2d0c5'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'USER_NOT_FOUND', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e4e3a2eb-5e88-4208-aea7-a7a90af2bfcc', 'message': '当前账号不存在'}, 'sheet_name': '登录模块', 'row': 3}
[ 2024-01-04 00:46:49,692,692 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:49,786,786 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'USER_LOGIN_INFO_NOT_FOUND', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': 'a5bdec96-1133-46cb-aaca-95b0bc79ab66', 'message': '当前账号不存在'}
[ 2024-01-04 00:46:49,787,787 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 当前账号不存在 == 当前账号不存在
[ 2024-01-04 00:46:49,788,788 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:49,788,788 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login0]
[ 2024-01-04 00:46:49,789,789 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <3> 行
[ 2024-01-04 00:46:49,789,789 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:46:49,789,789 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:46:49,789,789 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi288', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '9369', 'verifyId': '48d6e1625cda496280a5f69f1ac0a64f'}
[ 2024-01-04 00:46:49,789,789 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:46:49,789,789 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:46:49,790,790 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.09512589999999976
[ 2024-01-04 00:46:49,790,790 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯* 
[ 2024-01-04 00:46:49,792,792 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:49,798,798 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login002', 'title': '正确的用户名，错误的密码', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a1234567', 'verifyCode': '8844.43', 'verifyId': 'f1542b139f974437add53474062c9df6'}, 'expected': '登录失败，返回错误信息', 'expected_response': {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'yl-yft-api', 'traceId': 'e5665ff5-e379-4619-9467-175944d71239', 'message': '密码错误,输入密码不匹配'}, 'sheet_name': '登录模块', 'row': 4}
[ 2024-01-04 00:46:49,879,879 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:49,978,978 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'PASSWORD_ERROR', 'result': None, 'application': 'user-api|yl-yft-api', 'traceId': '7976e8b8-75f3-44ac-a043-d1b517c26438', 'message': '密码错误,输入密码不匹配'}
[ 2024-01-04 00:46:49,979,979 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : 密码错误,输入密码不匹配 == 密码错误,输入密码不匹配
[ 2024-01-04 00:46:49,979,979 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:49,979,979 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login1]
[ 2024-01-04 00:46:49,979,979 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <4> 行
[ 2024-01-04 00:46:49,979,979 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:46:49,979,979 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:46:49,980,980 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': '5690dddfa28ae085d23518a035707282', 'verifyCode': '6955', 'verifyId': '90ddd61b1b364a808d8162b549bec640'}
[ 2024-01-04 00:46:49,980,980 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:46:49,980,980 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:46:49,980,980 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.09947289999999942
[ 2024-01-04 00:46:49,980,980 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯* 
[ 2024-01-04 00:46:49,982,982 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:49,985,985 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 26 / login ] -- -> login参数化, 读取到数据: {'case_id': 'Login003', 'title': '用户名正确，密码正确', 'url': '/user/agency_user/username_pwd/sign_in', 'method': 'post', 'data': {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'a123456', 'verifyCode': '8844.43', 'verifyId': 'f56cd0d2ac32489cb5d0d0d2a2422558'}, 'expected': '登录成功，返回正确信息', 'expected_response': {'code': 'OK', 'result': {'userInfo': {'id': '1345922883512320001', 'createBy': '9527|超级管理员', 'createTime': 1609727996000, 'updateBy': '9527|超级管理员', 'updateTime': 1655706547000, 'remark': None, 'appId': '20210413170234896469661408411', 'agencyId': '9527', 'agencyCode': 'SUPER_9527', 'userType': 'SUPER_USER', 'loginAccount': 'ceshi2', 'userLevel': 'NORMAL', 'realName': '王佳琪', 'nickName': '王佳琪', 'pyRealName': 'WANGJIAQI', 'pytopRealName': 'W', 'sex': 'M', 'email': '2@qq.com', 'phone': '18629435474', 'birthday': 1655395200000, 'avatar': 'https://yl-prescription-share.oss-cn-beijing.aliyuncs.com/beta/SUPER_USER/20220620/519c09afaeb048caaf0f4e76166dfd18.jpg', 'idCard': '610121199802067410', 'provinceCode': None, 'cityCode': None, 'districtCode': None, 'address': None, 'registerDate': 1609727996000, 'loginDate': 1658540705000, 'lockStatus': 'N', 'deleteStatus': 'N', 'addressVo': None, 'agencyType': 'SUPER', 'orgId': '9527'}, 'loginId': 'ceshi2', 'lockStatus': 'N', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'token': 'SUPER_USER:83Ru5wCxDWIHHUrkiyU//t9ikw3xIjrecIXE/mqdKoI=0b64f647f9de4900aedc0902a60932d3'}, 'application': 'yl-yft-api', 'traceId': '6b61d2a7-0252-4b65-b50f-d85ae2168608', 'message': 'OK'}, 'sheet_name': '登录模块', 'row': 5}
[ 2024-01-04 00:46:50,063,063 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:50,195,195 ]  [ INFO     ]  [ MainThread  ]  [ test_login.py / 25 / test_login ] -- -> 响应结果: {'code': 'OK', 'result': {'userInfo': None, 'loginId': 'ceshi2', 'isDefaultPassword': 'Y', 'expire': '10800', 'userType': 'SUPER_USER', 'userName': '王佳琪', 'userId': '1345922883512320001', 'agencyType': 'SUPER', 'token': 'SUPER_USER:7qtoSKHKhYmMlVv9366DCDBT32hMWFklslrj4IbBugg=2cf096ad81644a9398766181e1bbfd63'}, 'application': 'yl-yft-api', 'traceId': '7f67cc5a3a494a4ea7a0cf85021f48ea', 'message': 'OK'}
[ 2024-01-04 00:46:50,196,196 ]  [ INFO     ]  [ MainThread  ]  [ base_assert.py / 112 / assert_common ] -- -> 断言通过 : OK == OK
[ 2024-01-04 00:46:50,196,196 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
[ 2024-01-04 00:46:50,196,196 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 40 / pytest_runtest_makereport ] -- -> 用例位置 : test_cases/test_login/test_login.py::TestLogin::test_login[login2]
[ 2024-01-04 00:46:50,197,197 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 43 / pytest_runtest_makereport ] -- -> 数据位置 : Sheet: <登录模块> 第 row: <5> 行
[ 2024-01-04 00:46:50,197,197 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 44 / pytest_runtest_makereport ] -- -> 请求地址 : /user/agency_user/username_pwd/sign_in
[ 2024-01-04 00:46:50,197,197 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 45 / pytest_runtest_makereport ] -- -> 请求方法 : post
[ 2024-01-04 00:46:50,197,197 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 46 / pytest_runtest_makereport ] -- -> 请求参数 : {'authType': 'usernamePassword', 'idType': 'USERNAME', 'loginId': 'ceshi2', 'password': 'dc483e80a7a0bd9ef71d8cf973673924', 'verifyCode': '5355', 'verifyId': '0363337fd4bc44aaba3028879f27650d'}
[ 2024-01-04 00:46:50,197,197 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 47 / pytest_runtest_makereport ] -- -> 测试结果 : passed
[ 2024-01-04 00:46:50,198,198 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 48 / pytest_runtest_makereport ] -- -> 异常信息 : None
[ 2024-01-04 00:46:50,198,198 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 49 / pytest_runtest_makereport ] -- -> 用例耗时 : 0.13338129999999992
[ 2024-01-04 00:46:50,198,198 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 50 / pytest_runtest_makereport ] -- ->  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯*  *¯* 
[ 2024-01-04 00:46:50,200,200 ]  [ INFO     ]  [ MainThread  ]  [ conftest.py / 36 / pytest_runtest_makereport ] -- ->  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_*  *_* 
